eng/common/scripts/Add-Issue-Comment.ps1

```diff
@@ -23,12 +23,7 @@ param(
 )
 
 . "${PSScriptRoot}\logging.ps1"
-
-$headers = @{
-  Authorization = "bearer $AuthToken"
-}
-
-$apiUrl = "https://api.github.com/repos/$RepoOwner/$RepoName/issues/$IssueNumber/comments"
+. "${PSScriptRoot}\Invoke-GitHub-API.ps1" -AuthToken $AuthToken
 
 $commentPrefixValue = [System.Environment]::GetEnvironmentVariable($CommentPrefix)
```

Comment:
Why are these reading from environment variables as opposed to the passed in variables? I would not have guess that this parameter the name of an environment variable.

Reply:
If someone wants to pass in an environment variable they should let the caller do that.

Reply:
I'm also not sure there is much value in splitting this into 3 parameters and instead suggest just one parameter for the comment and let the caller concat if they like.